api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,24 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Global target percentage of requests that will be force traced if the *x-client-trace-id*
+    // header is set. Percent is resolved to the nearest 1% (rounded down). Defaults to 1 (i.e.,
+    // 100%). This variable is a direct analog for the variable of the same name in the :ref:`HTTP
+    // Connection Manager <config_http_conn_man_runtime>`.
+    Percent client_enabled = 3;
+
+    // Global target percentage of requests that will be traced after all other checks have been
```

Comment:
The concept of "after all other checks" is questionable here. In a distributed system, we can not assume there is a fixed order of checks. It doesn't seem proper to assume tracing has the privilege of being the last decider.

Reply:
My understanding is that this is per-HTTP connection manager, which is not distributed, so I think we should just get rid of the "global" and call that out specifically.